R/app-driver-start.R

```diff
@@ -98,7 +98,8 @@ app_start_shiny <- function(
 
     if (i == max_i) {
       app_abort(self, private, paste0(
-        "Cannot find shiny port number. Error lines found:\n",
+        "Reached `load_timeout` limit and shiny app is still not running \n",
```

Comment:
I think it'd be helpful to say how long we waited in seconds and to provide some guidance on how to adjust `load_timeout` (if possible).